CHANGELOG.md

```diff
@@ -216,55 +216,181 @@ things. We want them to know that we have a lot of talent as well"
 
 This release wouldn't have been possible without the wonderful contributors
 to the [zero-to-jupyterhub](https://github.com/jupyterhub/zero-to-jupyterhub-k8s),
-[JupyterHub](https://github.com/jupyterhub/jupyterhub) and [KubeSpawner](https://github.com/jupyterhub/kubespawner)
+[JupyterHub](https://github.com/jupyterhub/jupyterhub), [KubeSpawner](https://github.com/jupyterhub/kubespawner)
+and [OAuthenticator](http://github.com/jupyterhub/oauthenticator) repos.
```

Comment:
There are two `repos.` See following line

Reply:
good catch! Fixed it! :)